ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-3node-ranGen.yaml

```diff
@@ -34,6 +34,11 @@ spec:
       metadata:
         name: openshift-node-performance-profile
       spec:
+        additionalKernelArgs:
+          - "idle=poll"
+          - "rcupdate.rcu_normal_after_boot=0"
+          - "kernel.timer_migration=1"
```

Comment:
Unfortunately tuned will overwrite this when it starts up, so we will need to keep the TunedPerformancePatch for sysctl
[sysctl]
              kernel.timer_migration=1

The updated TunedPerformancePatch will not require a reboot when applied.